offsets/offsets.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+
+
+def abbott_elec():
+	"""
+	This function returns the price per kWh at APP
+
+	Returns:
+	--------
+	per_kwh : float
+		The price in dollars per kWh for electricity produced
+		by APP
+	"""
+	per_kwh = 0.08  # [$/kWh]
```

Comment:
Can you add, in the comments maybe or in the docstring, a note regarding where the source came from.